src/builders/cypress/cypress-builder-options.ts

```diff
@@ -2,6 +2,7 @@ import { JsonObject } from '@angular-devkit/core';
 
 export interface CypressBuilderOptions extends JsonObject {
   baseUrl: string;
+  configPath: string;
```

Comment:
This appears to be working well, but I'm wondering if the option should be `configFile` to stay consistent with the Cypress docs language

Reply:
We should probably allow the user to pass in `false` to disable the use of a configuration file too (https://docs.cypress.io/guides/guides/command-line.html#cypress-run-config-file-lt-config-file-gt)

Reply:
Good points, I implemented them in the last commit (3150156).